### ISSUE TITLE
IOS not working

### ISSUE THREAD

**Issue by vinodlee** [gdagitrep/native_screenshot_ext#1]
I have tried to take a screenshot on the ios device, But int the IOS Platform return an empty path only. Is there any solution please help me with this.

**Comment by gdagitrep** [comment]
Yeah. I need to fix the iOS functionality. Feel free to create pull request
if you figure it out.
Thanks

**Comment by gdagitrep** [comment]
Fixed with https://github.com/gdagitrep/native_screenshot_ext/commit/62803d27b6b844b99b427c3465ca110d031fb238
